tox.ini

```diff
@@ -9,8 +9,18 @@ envlist =
     pypy3
 
 [testenv]
-usedevelop = true
 commands =
-    python setup.py -q test -q
+    zope-testrunner --test-path=src []
 deps =
     .[test]
+
+[testenv:coverage]
+usedevelop = true
+basepython =
+    python3.6
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage report --fail-under=67
```

Comment:
You promised 100% coverage in the PR title ;)

Reply:
Whoops, forgot to update that. Thanks.